app/policies/target_policy.rb

```diff
@@ -0,0 +1,5 @@
+class TargetPolicy < ApplicationPolicy
+  def create?
+    true
```

Comment:
Sincerely, I included it because if it is not there, the user can’t create the target because there are a application policy class that have this method  
`def create?`
` false`
  `end`.
Anyway, I think that this can be useful in the future if a functionality that restricts the number of targets a user can create is added.